bitnami/prometheus-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.34.0
 description: The Prometheus Operator for Kubernetes provides easy monitoring definitions for Kubernetes services and deployment and management of Prometheus instances.
 name: prometheus-operator
-version: 0.7.6
+version: 0.7.7
```

Comment:
When we add new parameters to our charts we bump the minor version. Please bump the minor version.